src/fonts/untitled-font-6.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg">
+<metadata>Generated by Fontastic.me</metadata>
+<defs>
+<font id="untitled-font-6" horiz-adv-x="512">
```

Comment:
I don't like the name. Could you please give this font, and other files (like `.eot`, `.ttf` and `.woff`) a good name? If they contain glyphs (icons) so call it that way.

Reply:
Due to browsers wars etc etc, it was a good practice to add all this font formats to font-face so the single font looked similarly in all browsers. But if our target is only the latest versions we can use only ```.woff``` and remove others.